components/src/atoms/price/Price.stories.js

```diff
@@ -0,0 +1,74 @@
+import { storiesOf } from '@storybook/vue'
+
+import APrice from './Price.vue'
+
+const info = `
+  ---
+  #### Use these modifiers with \`.price\` class.
+  - \`.price--large\` - Selector for use large size of price
+  ---
+`
+
+const defaultData = {
+  components: { APrice },
+  data: () => {
+    return {
+      price: '$1 400.00',
+      special: '$1 299 00',
+      old: '$1 999.99'
+    }
+  }
+}
+
+storiesOf('Atoms/Price', module)
+  .addParameters({ info })
+  .add('Default', () => ({
+    ...defaultData,
+    template: `
+      <a-price>
+        <span>{{ price }}</span>
+      </a-price>
+    `
+  }))
+  .add('Default -- Large', () => ({
```

Comment:
I think we don't need large option, It's stricte related with alpaca theme, right?